lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -192,6 +192,19 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     element.appendChild(passedElem);
     return element;
   }
+
+  /**
+   * @param {LH.ReportResult.Category} category
+   * @return {null|string}
+   * @override
+   */
+  static getLastScreenshot(category) {
```

Comment:
maybe we don't want this kind of api on a category renderer? Could it live in utils.js?

Reply:
> it could live in Util, sure. But it seems more specific to perf.

it is specific to perf, but it's also a utility method and doesn't really match the rest of the API (which is entirely construction and `render()` right now). It feels weird to mix DOM generation and helpful utility methods. And since it's static and not called from within `performance-category-renderer.js`, you have to know to call it with the perf category anyways. We also have other single-use methods in `utils.js`, like `getEnvironmentDisplayValues` or `getEmulationDescriptions`.

Though now that this is an audit, maybe we shouldn't have a method at all and the PSI renderer can just reach into the audit itself?